localmassifreader.go

```diff
@@ -0,0 +1,310 @@
+package veracity
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/datatrails/go-datatrails-common/azblob"
+	"github.com/datatrails/go-datatrails-common/logger"
+	"github.com/datatrails/go-datatrails-merklelog/massifs"
+)
+
+type Opener interface {
+	Open(string) (io.ReadCloser, error)
+}
+
+type DirLister interface {
+	// ListFiles returns list of absolute paths
+	// to files (not subdirectories) in a directory
+	ListFiles(string) ([]string, error)
+}
+
+type LocalMassifReader struct {
```

Comment:
awesome :) i love that we have a local massif reader now, is it worth putting it in the massifs module?

Reply:
I was thinking about that but the interface is kind of rough as we still expose the azure blob stuff quite a bit - I think once we have time to maybe review the interfaces on both ends we can refactor and pull this one into the massifs module